### PR TITLE
first draft of GRC data model narrative

### PR DESCRIPTION
**Pull request by tanggk**
first draft of GRC data model narrative

### CODE REVIEW

**Comment by tanggk** [comment]
Screen Shot 2021-03-02 at 1 28 49 PM
^this is what I see in VS Code for both the actual markdown + the rendering previewer

File: docs/data-model/org-grc.md [hunk 1]
```diff
@@ -1,3 +1,66 @@
...
+### Roughly, you can almost think of an organization’s policies, procedures, and controls to loosely align to the compliance or regulatory standards, sections, and requirements.
+
+**Policies** are high-level statements of management intent; they are written
+security documents which frequently satisfy external requirements. 
+
+    Examples of policies include: access management policies, data protection policies, human resource policies.
+
+> **Policies** are _implemented_ by **procedures**. 
+
+**Procedures** are written security documents which describe how to implement
```
Comment (a-u-h-g) [inline docs/data-model/org-grc.md:32]: would also mention how procedures relate to requirements

File: docs/data-model/org-grc.md [hunk 2]
```diff
@@ -1,3 +1,66 @@
 # JupiterOne data model for Governance, Risk, and Compliance (GRC)
 
-<iframe src="https://my.mindnode.com/2ayndotqvjEJ3qAeEfwyy6sgkRciyxRvH1sgpYw4/em#166,242,-3" frameborder="0" marginheight="0" marginwidth="0" style="border: 1px solid rgb(204, 204, 204); width: 720px; height: 720px;" onmousewheel=""></iframe>
\ No newline at end of file
+<iframe src="https://my.mindnode.com/2ayndotqvjEJ3qAeEfwyy6sgkRciyxRvH1sgpYw4/em#166,242,-3" frameborder="0" marginheight="0" marginwidth="0" style="border: 1px solid rgb(204, 204, 204); width: 720px; height: 720px;" onmousewheel=""></iframe>
```
Comment (a-u-h-g) [inline docs/data-model/org-grc.md:3]:
```suggestion
<iframe src="https://my.mindnode.com/2ayndotqvjEJ3qAeEfwyy6sgkRciyxRvH1sgpYw4/em#333.4,207.2,-2" frameborder="0" marginheight="0" marginwidth="0" style="border: 1px solid rgb(204, 204, 204); width: 720px; height: 720px;" onmousewheel=""></iframe>
```

File: docs/data-model/org-grc.md [hunk 3]
```diff
@@ -1,3 +1,66 @@
...
+> **control**.
+
+**Control Policies** or **configurations** are the technical settings whereby
+controls are implemented. 
+
+    Examples of control policies or configurations include: requiring 12+ characters including a number + a symbol for all passwords, using AES-256 cipher for encryption at rest, and for background checks, specifically looking at federal, criminal, state, county, city, financial, education checks.
+
+> **Vendors** _provide_ **controls**. 
+
+Vendors are frequently companies, organizations, or people.
```
Comment (a-u-h-g) [inline docs/data-model/org-grc.md:57]:
```suggestion
**Vendors** are frequently companies, organizations, or people. 
```

Comment (a-u-h-g) [reply docs/data-model/org-grc.md:57]: would also mention how `Vendors` can `PROVIDE` a `Control` to give context for the continued examples after

File: docs/data-model/org-grc.md [hunk 4]
```diff
@@ -1,3 +1,66 @@
...
+
+> A **standard** _has_ one or more **sections**; a **section** _has_ one or more
+> **requirements**.
+
+**Requirements** make up sections of a standard. Individual requirements outline
+the specification that needs to be met. 
+
+    Examples of requirements include: HIPAA §164.310(a)(1)(i), ISO 27001 6.1.3 a, PCI-DSS 8.4.b, or FedRAMP AC-2 (7).
+
+### Roughly, you can almost think of an organization’s policies, procedures, and controls to loosely align to the compliance or regulatory standards, sections, and requirements.
```
Comment (a-u-h-g) [inline docs/data-model/org-grc.md:23]: This seems too long and casual a statement for a heading. You could be better off having two H2 headings one for `## Standards, Sections, and Requirements` at the top, and one for `## Policies, Procedures, Controls, and Configurations` here. You can then put this sentence in bold immediately under the heading. You could also use H3 for all the Classes that you currently have in bold.

File: docs/data-model/org-grc.md (hunk 3 above)
Comment (tanggk) [reply docs/data-model/org-grc.md:57]: I updated

File: docs/data-model/org-grc.md (hunk 4 above)
Comment (tanggk) [reply docs/data-model/org-grc.md:23]: I updated

File: docs/data-model/org-grc.md [hunk 5]
```diff
@@ -1,3 +1,71 @@
...
+> **procedures**. 
+
+**Controls** are the technical, administrative, and physical safeguards that
+enforce the procedures; they can manifest commonly as a process managed by a
+person/team or as a product/service provided by a vendor. 
+
+    Examples of controls include: password settings, data encryption at rest or in transit,  and pre-employment background checks.
+
+> A **controlPolicy** or **configuration** _enforces_ or _manages_ a
+> **control**.
```
Comment (erkangz) [inline docs/data-model/org-grc.md:53]:
```suggestion
> A `ControlPolicy` or `Configuration` _enforces_ or _manages_ a
> `Control`.
```

File: docs/data-model/org-grc.md [hunk 6]
```diff
@@ -1,3 +1,71 @@
...
+    Examples of procedures include procedures around: password management, protecting data at rest, and employee screening.
+
+> **Policies** are _implemented_ by **procedures**; **controls** _implement_
+> **procedures**. 
+
+**Controls** are the technical, administrative, and physical safeguards that
+enforce the procedures; they can manifest commonly as a process managed by a
+person/team or as a product/service provided by a vendor. 
+
+    Examples of controls include: password settings, data encryption at rest or in transit,  and pre-employment background checks.
```
Comment (erkangz) [inline docs/data-model/org-grc.md:50]:
```suggestion
Examples of controls include: 

- user identity management, access control, multi-factor authentication
- data encryption at rest or in transit
- penetration testing, code scanning
- pre-employment background checks.
```

File: docs/data-model/org-grc.md [hunk 7]
```diff
@@ -1,3 +1,71 @@
...
+**Control Policies** or **configurations** are the technical settings whereby
+controls are implemented. 
+
+    Examples of control policies or configurations include: requiring 12+ characters including a number + a symbol for all passwords, using AES-256 cipher for encryption at rest, and for background checks, specifically looking at federal, criminal, state, county, city, financial, education checks.
+
+> **Vendors** _provide_ **controls**. 
+
+Vendors are frequently companies, organizations, or people that provide the controls
+
+    Examples of vendors include Microsoft Active Directory’s (AD) Group Policy Objects (GPO) for password requirements, Amazon Web Services’ (AWS) Key Management Service (KMS) for encryption at rest, or Checkr for background screens.
```
Comment (erkangz) [inline docs/data-model/org-grc.md:64]:
```suggestion
Examples of vendors include:

- Microsoft (the Vendor) for Active Directory (AD), user authentication, and access control
- Amazon Web Services (AWS, the Vendor) for Key Management Service (KMS)
- Checkr (the Vendor) for background screens
```

File: docs/data-model/org-grc.md [hunk 8]
```diff
@@ -1,3 +1,71 @@
...
+
+    Examples of controls include: password settings, data encryption at rest or in transit,  and pre-employment background checks.
+
+> A **controlPolicy** or **configuration** _enforces_ or _manages_ a
+> **control**.
+
+**Control Policies** or **configurations** are the technical settings whereby
+controls are implemented. 
+
+    Examples of control policies or configurations include: requiring 12+ characters including a number + a symbol for all passwords, using AES-256 cipher for encryption at rest, and for background checks, specifically looking at federal, criminal, state, county, city, financial, education checks.
```
Comment (erkangz) [inline docs/data-model/org-grc.md:58]:
```suggestion
Examples of control policies or configurations include: 

- requiring 12+ characters including a number + a symbol for all passwords
- using AES-256 cipher for encryption at rest
- for background checks, specifically include searches for federal, criminal, state, county, city, financial, and education verification
```

File: docs/data-model/org-grc.md [hunk 9]
```diff
@@ -1,3 +1,71 @@
...
+
+    Examples of policies include: access management policies, data protection policies, human resource policies.
+
+> **Policies** are _implemented_ by **procedures**. 
+
+**Procedures** are written security documents which describe how to implement
+policies via processes; the ‘who’, ‘what’, ‘when’, ‘how’, etc; they can be
+thought of as control or process descriptions. 
+
+    Examples of procedures include procedures around: password management, protecting data at rest, and employee screening.
```
Comment (erkangz) [inline docs/data-model/org-grc.md:41]:
```suggestion
Examples of procedures (aka control/process descriptions) include: 

- password management
- protecting data at rest 
- employee screening
```

**Review by erkangz — CHANGES_REQUESTED** [review]

File: docs/data-model/org-grc.md [hunk 10]
```diff
@@ -1,3 +1,71 @@
...
+    Examples of sections include: HIPAA Physical Safeguards (§164.310), ISO 27001 Clause 6, PCI-DSS Requirement 8, or the Access Control (AC) Family within FedRAMP.
+
+> A **standard** _has_ one or more **sections**; a **section** _has_ one or more
+> **requirements**.
+
+**Requirements** make up sections of a standard. Individual requirements outline
+the specification that needs to be met. 
+
+    Examples of requirements include: HIPAA §164.310(a)(1)(i), ISO 27001 6.1.3 a, PCI-DSS 8.4.b, or FedRAMP AC-2 (7).
+
```
[the remark this answers is not in the document]
Comment (tanggk) [reply docs/data-model/org-grc.md:25]: do you mean we need to build these 2x new data types/entities into the GRC model in the future? I'm not sure I follow 100%

[the remark this answers is not in the document]
Comment (tanggk) [reply docs/data-model/org-grc.md:25]: domains + controls can be defined by a standard, e.g., CIS; J1 has 22 domains + 100+ controls via P+P. CIS has 20 domains 127 controls

domain/controls can be defined by industry standards, or P+P defined by J1/your org.

[the remark this answers is not in the document]
Comment (tanggk) [reply docs/data-model/org-grc.md:31]: updated per comments

**Review by erkangz — APPROVED** [review]

**Pull request merged**